snappy/click_test.go

```diff
@@ -397,15 +379,14 @@ export HOME="$SNAP_USER_DATA"
 
 # Snap name is: pastebinit
 # App name is: pastebinit
-# Developer name is: mvo
 
-ubuntu-core-launcher pastebinit.pastebinit pastebinit.mvo_pastebinit_1.4.0.0.1 /snaps/pastebinit.mvo/1.4.0.0.1/bin/pastebinit "$@"
+ubuntu-core-launcher pastebinit.pastebinit pastebinit_pastebinit_1.4.0.0.1 /snaps/pastebinit/1.4.0.0.1/bin/pastebinit "$@"
```

Comment:
I strongly suspect this requires changes to the launcher but I'll let @jdstrand comment on this.